src/resources/Clusters/Nodes/ClusterNode.ts

```diff
@@ -0,0 +1,32 @@
+import API from '../../../APICore';
+import Resource from '../../Resource';
+import {ClusterNodeUpgradeDataModel} from '../ClusterInterfaces';
+import {ClusterNodeModel, ClusterNodeUpgradeModel} from './ClusterNodeInterfaces';
+
+export default class ClusterNode extends Resource {
+    static getBaseUrl = (id: string) => `/rest/organizations/${API.orgPlaceholder}/clusters/${id}/nodes`;
+
+    list(clusterId: string) {
+        return this.api.get<ClusterNodeModel[]>(ClusterNode.getBaseUrl(clusterId));
+    }
+
+    listUpgrades(clusterId: string, id: string) {
+        return this.api.get<ClusterNodeUpgradeModel[]>(`${ClusterNode.getBaseUrl(clusterId)}/${id}/upgrades`);
+    }
+
+    start(clusterId: string, id: string) {
+        return this.api.post<{}>(`${ClusterNode.getBaseUrl(clusterId)}/${id}/start`, {});
```

Comment:
We could add `{}` as default value for the second parameter of the `api.post` method. That way we would get
```ts
return this.api.post(`${ClusterNode.getBaseUrl(clusterId)}/${id}/start`);
```